src/sass/file-upload/stories/file-upload-cards.stories.js

```diff
@@ -0,0 +1,161 @@
+import {
+    uploading as uploadingCard,
+    uploadedPortraitImage as uploadedPortraitImageCard,
+    uploadedLandscapeImage as uploadedLandscapeImageCard,
+    uploadedVideo as uploadedVideoCard,
+    uploadedDocumentWithTitle as uploadedDocumentWithTitleCard,
+    uploadedDocumentWithTitleAndDescription as uploadedDocumentWithTitleAndDescriptionCard,
+    uploadedWithOverflow as uploadedWithOverflowCard,
+} from "../../card/stories/card.stories";
```

Comment:
Interesting. 
This is possibly a good pattern we can use in the future!

Reply:
I noticed a lot of repetition so reused the markup from other stories. I tried this with the app but it seems like it there may be some special logic with the highlight-code component that doesn't process the functions